src/vivarium_csu_ltbi/tools/builder.py

```diff
@@ -15,6 +16,12 @@
 DEFAULT_PATH = gbd.ARTIFACT_FOLDER / PROJ_NAME
 
 
+def set_to_known_value(df, set_to):
+    for col in df.columns:
```

Comment:
Don't need a separate method for this.  It's a one-liner: `df.loc[:, :] = set_to`